### DISCUSSION TITLE
Use single commit for the simple updates of dependencies

### DISCUSSION THREAD

**Discussion by rtritto** [TexteaInc/json-viewer#184] [Ideas]
To not create dirty history in `main` branch, should be used a single commit for the updates of dependencies or other strategy.

## Some options:
- create a new branch and cherry-pick all dependabot commits, then merge it using a squash (single commit)
- enable dependabot on a new branch and then merge it using a squash (single commit)

**Comment by rtritto** [comment]
FYI @pionxzh @anthonyalayo @Himself65

**Comment by pionxzh** [comment]
Yes it quote annoying. I will see how others deal with this kind of problems.

**Comment by pionxzh** [comment]
I will try setup [renovate](https://github.com/apps/renovate) to reduce the noise.
